pallets/simple-nft/src/mock.rs

```diff
@@ -62,7 +62,7 @@ impl system::Config for Test {
 }
 
 parameter_types! {
-    pub const MaxMetadataCount: u32 = 3;
+    pub const MaxMetadataCount: u32 = 4;
```

Comment:
increased this so the test for all metadata types (now 4) in one token still worked